linux-bash/read/src/main/bash/read_inputs.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
```

Comment:
Could we maybe add a comment at the top here with some example usage? I tried for example but didn't see any output:
- `./read_inputs.sh default_read`

Reply:
i did a simple menu to clarify things out ... just run ./read_inputs.sh